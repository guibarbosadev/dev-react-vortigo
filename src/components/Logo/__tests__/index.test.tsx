import React from 'react';
import { Logo } from '..';
import { render } from '@testing-library/react';

describe('<Logo test case', () => {
    it('matches snapshot', () => {
        const { container } = render(<Logo />);

        expect(container).toMatchSnapshot();
    });

    it('has testid "logo"', () => {
        const { queryByTestId } = render(<Logo />);

        expect(queryByTestId('logo')).not.toBeNull();
    });
});
