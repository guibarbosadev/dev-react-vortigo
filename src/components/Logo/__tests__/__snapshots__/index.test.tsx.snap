// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo test case matches snapshot 1`] = `
<div>
  <div
    data-testid="logo"
  >
    <img
      alt="Pokemon"
      srcset="pokemon-logo.png 1x, pokemon-logo@2x.png 2x, pokemon-logo@3x.png 3x"
    />
    <img
      alt="finder"
      class="finder"
      srcset="finder.png, finder@2x.png, finder@3x.png 3x"
    />
  </div>
</div>
`;
